cmd/compute-domain-kubelet-plugin/computedomain.go

```diff
@@ -59,7 +59,8 @@ type ComputeDomainManager struct {
 }
 
 type ComputeDomainDaemonSettings struct {
-	manager         *ComputeDomainManager
+	manager *ComputeDomainManager
+	// CD UUID
```

Comment:
What is this comment meant to represent?

Reply:
The this string carries the the CD UUID (as opposed to the CD name).